### ISSUE TITLE
[Feature request] Auto-refresh

### ISSUE THREAD

**Issue by zeiphon** [covid-19-au/covid-19-au.github.io#11]
Great dashboard, can I just request an auto-refresh feature?

Ideally it'd be nice to have a toggle with a few options, e.g. 10/30/60 minutes.

**Comment by freddiewanah** [comment]
Thanks, will add the last updated time to it.

**Comment by zeiphon** [comment]
Does it currently automatically refresh already? Hourly?

**Comment by freddiewanah** [comment]
Yes, our data comes from the Australian website [here](https://github.com/covid-19-au/covid-19-au.github.io/blob/dev/reference/reference.md). So there might be some lag. We're also looking for some other reliable data source atm.

**Comment by zeiphon** [comment]
Just to clarify, I mean if the browser is left open, is there a js timer that triggers a new data pull, or is the data only retrieved on full page load/refresh?

**Comment by freddiewanah** [comment]
ohhh, I see. Currently, only retrieve when the full page refresh, I'll add your comments  to our todo list

**Comment by blisx264** [comment]
<head>
  <meta http-equiv="refresh" content="nnn_sec">
</head>

NOTE:: The value "refresh" should be used carefully, as it takes the control of a page away from the user. Using "refresh" will cause a failure in W3C's Web Content Accessibility Guidelines.
